src/cursor/cur_json.c

```diff
@@ -8,6 +8,8 @@
 
 #include "wt_internal.h"
 
+static const u_char hex[] = "0123456789abcdef";
+
```

Comment:
This is now the 7th place in the code we declare `static const ... hex`.  They're not all the same either, some are `static const char` and some are `static const u_char`.  It would be nice to have a single one.